numbat-wasm/www/index.html

```diff
@@ -33,7 +33,7 @@
       <div id="terminal"></div>
       <p class="links"><a href="https://numbat.dev/doc/">Documentation</a> &bull; <a href="https://numbat.dev/doc/example-numbat_syntax.html">Syntax reference</a> &bull; <a href="https://numbat.dev/doc/web-usage.html">Commands and key bindings</a> &bull; <a href="https://github.com/sharkdp/numbat">Source code</a></p>
     </div>
-    <a href="https://github.com/sharkdp/numbat" class="github-corner" aria-label="View source on Github">
+    <a target="_blank" rel="noopener noreferrer" href="https://github.com/sharkdp/numbat" class="github-corner" aria-label="View source on Github">
```

Comment:
`noopener` is standard for `target="_blank"` links. `noreferrer` is not needed.
```suggestion
    <a target="_blank" href="https://github.com/sharkdp/numbat" class="github-corner" aria-label="View source on Github">
```